### ISSUE TITLE
scrollSpy issue when parent is position: relative

### ISSUE THREAD

**Issue by weavermedia** [inorganik/countUp.js#281]
```
[✅] Bug

CountUp.js version: 2.1.0

```

## Description

When the counter element's parent is `position: relative` the scrollSpy function does not trigger when the element scrolls into the view - it triggers when the page is scrolled to the top.

For my needs the following code works to define `bottomOfEl` within the `scrollSpy` function.
Not sure if it's suitable for a PR though.

```
// bottomOfEl = c.offsetTop + c.offsetHeight;
rect = self.el.getBoundingClientRect();
bottomOfEl = rect.top + rect.height + window.pageYOffset;
```

**Comment by huersch** [comment]
I have the same problem, any plans to include this fix?

**Comment by inorganik** [comment]
@weavermedia I tested your fix and it seems to work well for when the parent is position relative or position static. Nice job! Would you like to PR it?

**Comment by inorganik** [comment]
Fixed in v2.3.0. Thanks @weavermedia !
